ts/Dashboards/Actions/Bindings.ts

```diff
@@ -226,17 +225,35 @@ class Bindings {
 
     public static getCell(idOrElement: string): Cell|undefined {
         const cell = Bindings.getGUIElement(idOrElement);
-        return cell instanceof Cell ? cell : void 0;
+
+        if (!cell) {
+            return;
+        }
+        let isCellType = cell?.getType() === 'cell';
+
+        return isCellType ? (cell as Cell) : void 0;
     }
 
     public static getRow(idOrElement: string): Row|undefined {
         const row = Bindings.getGUIElement(idOrElement);
-        return row instanceof Row ? row : void 0;
+
+        if (!row) {
```

Comment:
The `return;` returns `undefined` and `return isLayoutType ? (layout as Layout) : void 0;` also when the type does not exist. As a result, you can simplify it. You could check the type in the first condition and avoid the inline condition in the last return.